pulser/simulation/simulation.py

```diff
@@ -325,55 +463,125 @@ def get_hamiltonian(self, time: float) -> qutip.Qobj:
                              "greater than or equal to 0.")
         return self._hamiltonian(time/1000)  # Creates new Qutip.Qobj
 
+    def _reset_samples(self) -> None:
+        """Sets all samples to their default values.
+
+        Used when running several noisy runs, each one setting new samples.
+        """
+        self.samples = {addr: {basis: {}
+                               for basis in ['ground-rydberg', 'digital']}
+                        for addr in ['Global', 'Local']}
+
+    def _init_dephasing(self) -> None:
+        """Initializes dephasing collapse operators."""
+        if self.basis_name == 'digital' or self.basis_name == 'all':
+            raise ValueError("Cannot include dephasing noise in digital-"
```

Comment:
Is this impossible or not implemented? If not implemented, use `NotImplementedError`